diff --git a/cdroid_install_libs.sh b/cdroid_install_libs.sh
deleted file mode 100755
index 3bc59d0..0000000
--- a/cdroid_install_libs.sh
+++ /dev/null
@@ -1,39 +0,0 @@
-#处理参数，规范化参数
-ARGS=`getopt -a -o t:h:: --long triplet:,help:: -- "$@"`
-#echo $ARGS
-eval set -- "${ARGS}"
-VCPKGROOT=""
-#将规范化后的命令行参数分配至位置参数（$1,$2,...)
-while :
-do
-   case $1 in
-        -t|--triplet)
-                TRIPLET=$2
-                echo "triplet=$TRIPLET"
-                shift
-                ;;
-        -h|--help)
-                SHOWHELP=1
-                echo "showhelp"
-                shift
-                ;;
-        --)
-                shift
-                break
-                ;;
-        *)
-                break
-                ;;
-   esac
-   shift
-done
-
-if [ -n"$TRIPLET" ]; then
-   ./vcpkg install zlib libunibreak libpng[apng] giflib libwebp libjpeg-turbo[jpeg8] libzip[core] gtest jsoncpp freetype[zlib,bzip2] cairo --triplet=$TRIPLET
-   ./vcpkg install ghc-filesystem libqrencode fribidi curl[mbedtls] libhv --recurse --triplet=$TRIPLET
-else
-   echo $0" --triplet=triplet_name" 
-fi
-
-#skia without opengl/vulkan install params
-#./vcpkg install skia[core,fontconfig,freetype,harfbuzz]
diff --git a/cdroid_remove_libs.sh b/cdroid_remove_libs.sh
deleted file mode 100755
index 829a654..0000000
--- a/cdroid_remove_libs.sh
+++ /dev/null
@@ -1,36 +0,0 @@
-#处理参数，规范化参数
-ARGS=`getopt -a -o t:h:: --long triplet:,help:: -- "$@"`
-#echo $ARGS
-eval set -- "${ARGS}"
-VCPKGROOT=""
-#将规范化后的命令行参数分配至位置参数（$1,$2,...)
-while :
-do
-   case $1 in
-        -t|--triplet)
-                TRIPLET=$2
-                echo "triplet=$TRIPLET"
-                shift
-                ;;
-        -h|--help)
-                SHOWHELP=1
-                echo "showhelp"
-                shift
-                ;;
-        --)
-                shift
-                break
-                ;;
-        *)
-                break
-                ;;
-   esac
-   shift
-done
-
-if [ -n "$TRIPLET" ]; then
-   ./vcpkg remove bluez zlib libuuid libunibreak giflib libjpeg-turbo libwebp expat bzip2 libzip gtest jsoncpp lzo freetype cairo --triplet=$TRIPLET --recurse
-   ./vcpkg remove ghc-filesystem libqrencode fribidi curl mbedtls openssl libhv zint --triplet=$TRIPLET --recurse
-else
-   echo $0" --triplet=triplet_name"
-fi
diff --git a/scripts/toolchains/d211_meson_cross.txt b/scripts/toolchains/d211_meson_cross.txt
deleted file mode 100755
index 5a0eaf4..0000000
--- a/scripts/toolchains/d211_meson_cross.txt
+++ /dev/null
@@ -1,10 +0,0 @@
-[binaries]
-c = 'riscv64-unknown-linux-gnu-gcc'
-cpp = 'riscv64-unknown-linux-gnu-g++'
-ar = 'riscv64-unknown-linux-gnu-ar'
-
-[host_machine]
-system = 'linux'
-cpu_family = 'riscv64'
-cpu = 'riscv64'
-endian = 'little'
diff --git a/scripts/toolchains/hisi3536-toolchain.cmake b/scripts/toolchains/hisi3536-toolchain.cmake
deleted file mode 100755
index 88c9659..0000000
--- a/scripts/toolchains/hisi3536-toolchain.cmake
+++ /dev/null
@@ -1,11 +0,0 @@
-SET(CMAKE_SYSTEM_NAME Linux)
-
-SET(TOOLCHAIN_DIR /opt/hisi-linux/x86-arm/arm-hisiv500-linux/target)
-set(CMAKE_CROSSCOMPILING ON CACHE BOOL "")
-set(CMAKE_C_FLAGS "-std=gnu99")
-set(CMAKE_CXX_FLAGS "-std=c++11")
-SET(CMAKE_SYSTEM_PROCESSOR arm)
-SET(CMAKE_CXX_COMPILER ${TOOLCHAIN_DIR}/bin/arm-hisiv500-linux-g++)
-SET(CMAKE_C_COMPILER ${TOOLCHAIN_DIR}/bin/arm-hisiv500-linux-gcc)
-SET(CMAKE_ASM_COMPILER ${TOOLCHAIN_DIR}/bin/arm-hisiv500-linux-gcc)
-SET(CMAKE_FIND_ROOT_PATH ${TOOLCHAIN_DIR})#/arm-hisiv500-linux-uclibcgnueabi/lib)
diff --git a/scripts/toolchains/ingenic-toolchain.cmake b/scripts/toolchains/ingenic-toolchain.cmake
deleted file mode 100755
index 0fc560b..0000000
--- a/scripts/toolchains/ingenic-toolchain.cmake
+++ /dev/null
@@ -1,14 +0,0 @@
-
-SET(CMAKE_SYSTEM_NAME Linux)
-
-SET(TOOLCHAIN_DIR /opt/mips-ingenic-toolchain-720)
-#SET(TOOLCHAIN_DIR /opt/mips-ingenic-toolchain-930)
-#SET(TOOLCHAIN_DIR /opt/mips-ingenic-toolchain-1210)
-SET(CMAKE_CROSSCOMPILING true)
-SET(VCPKG_MESON_CROSS_FILE ${VCPKG_ROOT_DIR}/scripts/toolchains/ingeric_meson_cross.txt)
-
-SET(CMAKE_SYSTEM_PROCESSOR mips)
-SET(CMAKE_CXX_COMPILER ${TOOLCHAIN_DIR}/bin/mips-linux-gnu-g++)
-SET(CMAKE_C_COMPILER ${TOOLCHAIN_DIR}/bin/mips-linux-gnu-gcc)
-SET(CMAKE_ASM_COMPILER ${TOOLCHAIN_DIR}/bin/mips-linux-gnu-gcc)
-SET(CMAKE_FIND_ROOT_PATH ${TOOLCHAIN_DIR}/mips-linux-gnueabihf/libc)
diff --git a/scripts/toolchains/ingenic_meson_cross.txt b/scripts/toolchains/ingenic_meson_cross.txt
deleted file mode 100755
index 987b985..0000000
--- a/scripts/toolchains/ingenic_meson_cross.txt
+++ /dev/null
@@ -1,14 +0,0 @@
-[binaries]
-c = 'mips-linux-gnu-gcc'
-cpp = 'mips-linux-gnu-g++'
-ar = 'mips-linux-gnu-ar'
-
-[host_machine]
-system = 'linux'
-cpu_family = 'mips'
-cpu = 'mips'
-endian = 'little'
-
-host_cpu ='mips'
-host_cpu_fam = 'mips'
-
diff --git a/scripts/toolchains/r818-toolchain.cmake b/scripts/toolchains/r818-toolchain.cmake
deleted file mode 100755
index 95bbc08..0000000
--- a/scripts/toolchains/r818-toolchain.cmake
+++ /dev/null
@@ -1,11 +0,0 @@
-
-SET(CMAKE_SYSTEM_NAME Linux)
-
-SET(TOOLCHAIN_DIR /opt/r818-toolchain/gcc/linux-x86/aarch64/toolchain-sunxi-glibc/toolchain)
-set(CMAKE_CROSSCOMPILING true)
-
-SET(CMAKE_SYSTEM_PROCESSOR aarch64)
-SET(CMAKE_CXX_COMPILER ${TOOLCHAIN_DIR}/bin/aarch64-openwrt-linux-g++)
-SET(CMAKE_C_COMPILER ${TOOLCHAIN_DIR}/bin/aarch64-openwrt-linux-gcc)
-SET(CMAKE_ASM_COMPILER ${TOOLCHAIN_DIR}/bin/aarch64-openwrt-linux-gcc)
-SET(CMAKE_FIND_ROOT_PATH ${TOOLCHAIN_DIR})#/aarch64-openwrt-linux-gnu)
diff --git a/scripts/toolchains/riscv64-d211-toolchain.cmake b/scripts/toolchains/riscv64-d211-toolchain.cmake
deleted file mode 100755
index d4cf27d..0000000
--- a/scripts/toolchains/riscv64-d211-toolchain.cmake
+++ /dev/null
@@ -1,12 +0,0 @@
-
-SET(CMAKE_SYSTEM_NAME Linux)
-
-SET(TOOLCHAIN_DIR /opt/riscv64-linux-x86_64)
-SET(CMAKE_CROSSCOMPILING true)
-SET(VCPKG_MESON_CROSS_FILE ${VCPKG_ROOT_DIR}/scripts/toolchains/d211_meson_cross.txt)
-
-SET(CMAKE_SYSTEM_PROCESSOR riscv64)
-SET(CMAKE_CXX_COMPILER ${TOOLCHAIN_DIR}/bin/riscv64-unknown-linux-gnu-g++)
-SET(CMAKE_C_COMPILER ${TOOLCHAIN_DIR}/bin/riscv64-unknown-linux-gnu-gcc)
-SET(CMAKE_ASM_COMPILER ${TOOLCHAIN_DIR}/bin/riscv64-unknown-linux-gnu-gcc)
-SET(CMAKE_FIND_ROOT_PATH ${TOOLCHAIN_DIR}/sysroot)
diff --git a/scripts/toolchains/rk3506-toolchain.cmake b/scripts/toolchains/rk3506-toolchain.cmake
deleted file mode 100755
index 7ff6602..0000000
--- a/scripts/toolchains/rk3506-toolchain.cmake
+++ /dev/null
@@ -1,17 +0,0 @@
-SET(CMAKE_SYSTEM_NAME Linux)
-
-SET(TOOLCHAIN_DIR /opt/rk3506-arm-linux-gnueabihf)
-SET(CMAKE_CROSSCOMPILING true)
-
-SET(VCPKG_MESON_CROSS_FILE ${VCPKG_ROOT_DIR}/scripts/toolchains/rk3506_meson_cross.txt)
-SET(ENV{"STAGING_DIR"} ${TOOLCHAIN_DIR}/arm-buildroot-linux-gnueabihf)
-SET(CMAKE_SYSTEM_PROCESSOR arm)
-SET(CMAKE_CXX_COMPILER ${TOOLCHAIN_DIR}/bin/arm-buildroot-linux-gnueabihf-g++)
-SET(CMAKE_C_COMPILER ${TOOLCHAIN_DIR}/bin/arm-buildroot-linux-gnueabihf-gcc)
-SET(CMAKE_ASM_COMPILER ${TOOLCHAIN_DIR}/bin/arm-buildroot-linux-gnueabihf-gcc)
-SET(CMAKE_FIND_ROOT_PATH ${TOOLCHAIN_DIR}/arm-buildroot-linux-gnueabihf)
-
-IF(EXISTS ${TOOLCHAIN_DIR}/arm-buildroot-linux-gnueabihf/sysroot)
-  SET(ENV{PKG_CONFIG_PATH} "${TOOLCHAIN_DIR}/arm-buildroot-linux-gnueabihf/sysroot/usr/lib/pkgconfig:$ENV{PKG_CONFIG_PATH}")
-  SET(CMAKE_SYSROOT ${TOOLCHAIN_DIR}/arm-buildroot-linux-gnueabihf/sysroot)
-ENDIF()
\ No newline at end of file
diff --git a/scripts/toolchains/rk3506_meson_cross.txt b/scripts/toolchains/rk3506_meson_cross.txt
deleted file mode 100755
index 62d1e81..0000000
--- a/scripts/toolchains/rk3506_meson_cross.txt
+++ /dev/null
@@ -1,14 +0,0 @@
-[binaries]
-c = 'arm-buildroot-linux-gnueabihf-gcc'
-cpp = 'arm-buildroot-linux-gnueabihf-g++'
-ar = 'arm-buildroot-linux-gnueabihf-gcc-ar'
-
-[host_machine]
-system = 'linux'
-cpu_family = 'arm'
-cpu = 'arm'
-endian = 'little'
-
-host_cpu ='arm'
-host_cpu_fam = 'arm'
-
diff --git a/scripts/toolchains/ssd202-mtitoolchain.cmake b/scripts/toolchains/ssd202-mtitoolchain.cmake
deleted file mode 100755
index e273d9f..0000000
--- a/scripts/toolchains/ssd202-mtitoolchain.cmake
+++ /dev/null
@@ -1,17 +0,0 @@
-
-SET(CMAKE_SYSTEM_NAME Linux)
-
-SET(TOOLCHAIN_DIR /opt/sigma-arm-linux-gnueabihf/)
-SET(CMAKE_CROSSCOMPILING true)
-
-SET(CMAKE_SYSTEM_PROCESSOR arm)
-SET(CMAKE_CXX_COMPILER ${TOOLCHAIN_DIR}/bin/arm-linux-gnueabihf-g++)
-SET(CMAKE_C_COMPILER ${TOOLCHAIN_DIR}/bin/arm-linux-gnueabihf-gcc)
-SET(CMAKE_ASM_COMPILER ${TOOLCHAIN_DIR}/bin/arm-linux-gnueabihf-gcc)
-SET(CMAKE_FIND_ROOT_PATH ${TOOLCHAIN_DIR})
-SET(CMAKE_POSITION_INDEPENDENT_CODE ON)
-
-IF(EXISTS ${TOOLCHAIN_DIR}/sysroot)
-  SET(ENV{PKG_CONFIG_PATH} "${TOOLCHAIN_DIR}/sysroot//usr/lib/pkgconfig:$ENV{PKG_CONFIG_PATH}")
-  SET(CMAKE_SYSROOT ${TOOLCHAIN_DIR}/sysroot)
-ENDIF()
diff --git a/scripts/toolchains/tinat113-toolchain.cmake b/scripts/toolchains/tinat113-toolchain.cmake
deleted file mode 100755
index 017df6f..0000000
--- a/scripts/toolchains/tinat113-toolchain.cmake
+++ /dev/null
@@ -1,15 +0,0 @@
-
-SET(CMAKE_SYSTEM_NAME Linux)
-
-SET(TOOLCHAIN_DIR /opt/tinat113-arm-toolchain)
-SET(CMAKE_CROSSCOMPILING true)
-SET(VCPKG_MESON_CROSS_FILE ${VCPKG_ROOT_DIR}/scripts/toolchains/tinat113_meson_cross.txt)
-SET(ENV("STAGING_DIR") /opt/tinat113-arm-toolchain/target)
-SET(CMAKE_SYSTEM_PROCESSOR arm)
-SET(CMAKE_CXX_COMPILER ${TOOLCHAIN_DIR}/bin/arm-openwrt-linux-g++)
-SET(CMAKE_C_COMPILER ${TOOLCHAIN_DIR}/bin/arm-openwrt-linux-gcc)
-SET(CMAKE_ASM_COMPILER ${TOOLCHAIN_DIR}/bin/arm-openwrt-linux-gcc)
-SET(CMAKE_FIND_ROOT_PATH ${TOOLCHAIN_DIR}/target)#/arm-openwrt-linux-/libc)
-
-set(ENV{PKG_CONFIG_PATH} "${TOOLCHAIN_DIR}/target/usr/lib/pkgconfig:$ENV{PKG_CONFIG_PATH}")
-SET(CMAKE_SYSROOT ${TOOLCHAIN_DIR}/target)
diff --git a/scripts/toolchains/tinat113_meson_cross.txt b/scripts/toolchains/tinat113_meson_cross.txt
deleted file mode 100755
index 25348de..0000000
--- a/scripts/toolchains/tinat113_meson_cross.txt
+++ /dev/null
@@ -1,14 +0,0 @@
-[binaries]
-c = 'arm-openwrt-linux-gcc'
-cpp = 'arm-openwrt-linux-g++'
-ar = 'arm-openwrt-linux-ar'
-
-[host_machine]
-system = 'linux'
-cpu_family = 'arm'
-cpu = 'arm'
-endian = 'little'
-
-host_cpu ='arm'
-host_cpu_fam = 'arm'
-
diff --git a/triplets/community/hisi3536-linux.cmake b/triplets/community/hisi3536-linux.cmake
deleted file mode 100755
index 3adb4a1..0000000
--- a/triplets/community/hisi3536-linux.cmake
+++ /dev/null
@@ -1,8 +0,0 @@
-set(VCPKG_TARGET_ARCHITECTURE arm)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE dynamic)
-set(VCPKG_CMAKE_SYSTEM_NAME Linux)
-set(CDROID_PORT "hi3536")
-set(VCPKG_MAKE_BUILD_TRIPLET "--host=arm-hisiv500-linux-uclibcgnueabi")
-set(VCPKG_PLATFORM_TOOLSET "arm-hisiv500-linux-uclibcgnueabi")
-set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${VCPKG_ROOT_DIR}/scripts/toolchains/hisi3536-toolchain.cmake")
diff --git a/triplets/community/ingenic-linux-dynamic.cmake b/triplets/community/ingenic-linux-dynamic.cmake
deleted file mode 100755
index 5ba5250..0000000
--- a/triplets/community/ingenic-linux-dynamic.cmake
+++ /dev/null
@@ -1,12 +0,0 @@
-set(VCPKG_TARGET_ARCHITECTURE mips)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE dynamic)
-set(VCPKG_CROSSCOMPILING TRUE)
-set(VCPKG_MESON_CROSS_FILE "${VCPKG_ROOT_DIR}/scripts/toolchains/ingenic_meson_cross.txt")
-
-set(VCPKG_CMAKE_SYSTEM_NAME Linux)
-set(VCPKG_MAKE_BUILD_TRIPLET "--host=mips-linux-gnu")
-set(VCPKG_PLATFORM_TOOLSET "mips-linux-gnu")
-set(CDROID_PORT "ingenic")
-set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${VCPKG_ROOT_DIR}/scripts/toolchains/ingenic-toolchain.cmake")
-
diff --git a/triplets/community/ingenic-linux.cmake b/triplets/community/ingenic-linux.cmake
deleted file mode 100755
index 96ba134..0000000
--- a/triplets/community/ingenic-linux.cmake
+++ /dev/null
@@ -1,15 +0,0 @@
-set(VCPKG_TARGET_ARCHITECTURE mips)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE static)
-set(VCPKG_CROSSCOMPILING TRUE)
-set(VCPKG_MESON_CROSS_FILE "${VCPKG_ROOT_DIR}/scripts/toolchains/ingenic_meson_cross.txt")
-
-set(VCPKG_CMAKE_SYSTEM_OPTIONS "-DCMAKE_POSITION_INDEPENDENT_CODE=ON")
-set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-
-set(VCPKG_CMAKE_SYSTEM_NAME Linux)
-set(VCPKG_MAKE_BUILD_TRIPLET "--host=mips-linux-gnu")
-set(VCPKG_PLATFORM_TOOLSET "mips-linux-gnu")
-set(CDROID_PORT "ingenic")
-set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${VCPKG_ROOT_DIR}/scripts/toolchains/ingenic-toolchain.cmake")
-
diff --git a/triplets/community/r818-linux-dynamic.cmake b/triplets/community/r818-linux-dynamic.cmake
deleted file mode 100755
index 2e413ff..0000000
--- a/triplets/community/r818-linux-dynamic.cmake
+++ /dev/null
@@ -1,10 +0,0 @@
-set(VCPKG_TARGET_ARCHITECTURE arm64)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE dynamic)
-set(VCPKG_CMAKE_SYSTEM_NAME Linux)
-set(VCPKG_CROSSCOMPILING TRUE)
-set(VCPKG_BUILD_TYPE release)
-set(VCPKG_MAKE_BUILD_TRIPLET "--host=aarch64-oprnwrt-linux-gnu")
-set(VCPKG_PLATFORM_TOOLSET "aarch64-openwrt-linux-gnu")
-set(R818_TOOLCHAIN /opt/vcpkg/r818-toolchain.cmake)
-set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${R818_TOOLCHAIN}")
diff --git a/triplets/community/r818-linux.cmake b/triplets/community/r818-linux.cmake
deleted file mode 100755
index 7a9a051..0000000
--- a/triplets/community/r818-linux.cmake
+++ /dev/null
@@ -1,12 +0,0 @@
-set(VCPKG_TARGET_ARCHITECTURE arm64)
-set(VCPKG_CRT_LINKAGE static)
-set(VCPKG_LIBRARY_LINKAGE static)
-set(VCPKG_CMAKE_SYSTEM_NAME Linux)
-set(VCPKG_CROSSCOMPILING TRUE)
-set(VCPKG_BUILD_TYPE release)
-set(VCPKG_CMAKE_SYSTEM_OPTIONS "-DCMAKE_POSITION_INDEPENDENT_CODE=ON")
-set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-set(VCPKG_MAKE_BUILD_TRIPLET "--host=aarch64-openwrt-linux-gnu-")
-set(VCPKG_PLATFORM_TOOLSET "aarch64-openwrt-linux-gnu")
-
-set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${VCPKG_ROOT_DIR}/scripts/toolchains/r818-toolchain.cmake")
diff --git a/triplets/community/rk3506-linux-dynamic.cmake b/triplets/community/rk3506-linux-dynamic.cmake
deleted file mode 100755
index 07c7c33..0000000
--- a/triplets/community/rk3506-linux-dynamic.cmake
+++ /dev/null
@@ -1,10 +0,0 @@
-set(VCPKG_TARGET_ARCHITECTURE arm)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE dynamic)
-set(VCPKG_BUILD_TYPE release)
-set(VCPKG_CMAKE_SYSTEM_NAME Linux)
-set(VCPKG_MAKE_BUILD_TRIPLET "--host=arm-buildroot-linux-gnueabihf")
-set(VCPKG_PLATFORM_TOOLSET "arm-buildroot-linux-gnueabihf")
-set(CDROID_PORT "rk3506")
-set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${VCPKG_ROOT_DIR}/scripts/toolchains/rk3506-toolchain.cmake")
-
diff --git a/triplets/community/rk3506-linux.cmake b/triplets/community/rk3506-linux.cmake
deleted file mode 100755
index 602c9b4..0000000
--- a/triplets/community/rk3506-linux.cmake
+++ /dev/null
@@ -1,10 +0,0 @@
-set(VCPKG_TARGET_ARCHITECTURE arm)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE static)
-set(VCPKG_BUILD_TYPE release)
-set(VCPKG_CMAKE_SYSTEM_NAME Linux)
-set(VCPKG_MAKE_BUILD_TRIPLET "--host=arm-buildroot-linux-gnueabihf")
-set(VCPKG_PLATFORM_TOOLSET "arm-buildroot-linux-gnueabihf")
-set(CDROID_PORT "rk3506")
-set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${VCPKG_ROOT_DIR}/scripts/toolchains/rk3506-toolchain.cmake")
-
diff --git a/triplets/community/sigma-linux-dynamic.cmake b/triplets/community/sigma-linux-dynamic.cmake
deleted file mode 100755
index 17802d3..0000000
--- a/triplets/community/sigma-linux-dynamic.cmake
+++ /dev/null
@@ -1,11 +0,0 @@
-#for sigmastar ssd2xx (arm32)
-set(VCPKG_TARGET_ARCHITECTURE arm)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE dynamic)
-set(VCPKG_BUILD_TYPE release)
-set(VCPKG_CMAKE_SYSTEM_NAME Linux)
-set(VCPKG_MAKE_BUILD_TRIPLET "--host=arm-linux-gnueabihf")
-set(VCPKG_PLATFORM_TOOLSET "arm-linux-gnueabihf")
-set(CDROID_PORT "sigma")
-set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${VCPKG_ROOT_DIR}/scripts/toolchains/ssd202-mtitoolchain.cmake")
-
diff --git a/triplets/community/sigma-linux.cmake b/triplets/community/sigma-linux.cmake
deleted file mode 100755
index 23aa3ee..0000000
--- a/triplets/community/sigma-linux.cmake
+++ /dev/null
@@ -1,11 +0,0 @@
-set(VCPKG_TARGET_ARCHITECTURE arm)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE static)
-set(VCPKG_BUILD_TYPE release)
-set(VCPKG_BUILD_TYPE release)
-set(VCPKG_CMAKE_SYSTEM_NAME Linux)
-set(VCPKG_MAKE_BUILD_TRIPLET "--host=arm-linux-gnueabihf")
-set(VCPKG_PLATFORM_TOOLSET "arm-linux-gnueabihf")
-set(CDROID_PORT "sigma")
-set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${VCPKG_ROOT_DIR}/scripts/toolchains/ssd202-mtitoolchain.cmake")
-
diff --git a/triplets/community/tinat113-linux-dynamic.cmake b/triplets/community/tinat113-linux-dynamic.cmake
deleted file mode 100755
index 13137f7..0000000
--- a/triplets/community/tinat113-linux-dynamic.cmake
+++ /dev/null
@@ -1,10 +0,0 @@
-set(VCPKG_TARGET_ARCHITECTURE arm)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE dynamic)
-set(VCPKG_BUILD_TYPE release)
-set(VCPKG_CMAKE_SYSTEM_NAME Linux)
-set(VCPKG_MAKE_BUILD_TRIPLET "--host=arm-openwrt-linux")
-set(VCPKG_PLATFORM_TOOLSET "arm-openwrt-linux")
-set(CDROID_PORT "tinat113")
-set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${VCPKG_ROOT_DIR}/scripts/toolchains/tinat113-toolchain.cmake")
-
diff --git a/triplets/community/tinat113-linux.cmake b/triplets/community/tinat113-linux.cmake
deleted file mode 100755
index 74b3f5f..0000000
--- a/triplets/community/tinat113-linux.cmake
+++ /dev/null
@@ -1,10 +0,0 @@
-set(VCPKG_TARGET_ARCHITECTURE arm)
-set(VCPKG_CRT_LINKAGE dynamic)
-set(VCPKG_LIBRARY_LINKAGE static)
-set(VCPKG_BUILD_TYPE release)
-set(VCPKG_CMAKE_SYSTEM_NAME Linux)
-set(VCPKG_MAKE_BUILD_TRIPLET "--host=arm-openwrt-linux")
-set(VCPKG_PLATFORM_TOOLSET "arm-openwrt-linux")
-set(CDROID_PORT "tinat113")
-set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${VCPKG_ROOT_DIR}/scripts/toolchains/tinat113-toolchain.cmake")
-
